modules/client-configuration/pages/registration-cli.adoc

```diff
@@ -18,8 +18,9 @@ Use the bootstrap script procedure to register traditional clients.
 For more information, see xref:registration-bootstrap.adoc[].
 ====
 
-This procedure requires that you have installed the [package]``salt-minion`` package on the Salt client before registration.
+This procedure requires that you have installed the [package]``venv-salt-minion`` package (Salt bundle) on the Salt client before registration.
 For SLE 12 based clients, you also must have activated the [systemitem]``Advanced Systems Management`` module.
```

Comment:
Do we still have a requirement to have `Advanced Systems Management` activated for SLE 12 based clients?

Reply:
@juliogonzalez can you help with this question?

Reply:
@mcalmer in https://documentation.suse.com/suma/4.2/en/suse-manager/client-configuration/clients-sle.html now we only list SLE 15 clients.  Maybe, here we can also drop details about older clients such as SLE 12.  Is this correct, Michael?

Reply:
It's strange. SLE 12 SP4 general support ends - 31 Oct 2024, LTSS - 31 Oct 2027 and SLE 12 SP4 LTSS ends 30 Jun 2023.
Do we support bootsrapping the systems with LTSS? I think yes.

Reply:
> Do we still have a requirement to have Advanced Systems Management activated for SLE 12 based clients?

I am not sure I am understanding. Salt for SUSE Manager SLE12 clients comes from the client tools. `Advanced Systems Management` also provides it, for customers who do not use SUSE Manager.

> @mcalmer in https://documentation.suse.com/suma/4.2/en/suse-manager/client-configuration/clients-sle.html now we only list SLE 15 clients. Maybe, here we can also drop details about older clients such as SLE 12. Is this correct, Michael?

FMPOV https://documentation.suse.com/suma/4.2/en/suse-manager/client-configuration/clients-sle.html is wrong. We even support (for a few more months) SLE11.

> Do we support bootsrapping the systems with LTSS? I think yes.
Yes, we do support them at SUSE Manager.

Reply:
I'm not sure about the note that `Advanced Systems Management` must be activated on the system. I think it's not correct.